README.md

```diff
@@ -389,6 +390,23 @@ Running the build
     and you are behind a non-transparent proxy server,
     [configure your proxy server in Maven](http://maven.apache.org/settings.html#Proxies).
 
+ Note: if you get the following error: 
+ 
+    ```shell
+    [ERROR] Failed to execute goal org.codehaus.mojo:jaxb2-maven-plugin:1.5:schemagen (generate-xsd) on project kie-remote-jaxb-gen: Failed to generate schema: InvocationTargetException: Class not foundcom/sun/tools/apt/Main.class -> [Help 1]
+    ``` 
```

Comment:
oooh.. I have needed 6.2.x branch and that resolution is not present there. Maybe, is better to add that resolution also in that branch?